src/delphi/eval/calc_model_group_stats.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+
+
+def calc_model_group_stats(
+    tokenized_corpus_dataset: list,
+    logprob_datasets: dict[str, list[list[float]]],
+    token_groups: dict[int, dict[str, bool]],
+    token_labels: list[str],
+) -> dict[tuple[str, str], dict[str, float]]:
```

Comment:
we should probably start using more dataclasses

Reply:
Strong endorse